spa_ui/self_service/bower.json

```diff
@@ -30,27 +30,23 @@
     "angular-messages": "~1.3.0",
     "angular-ui-router": "~0.2.0",
     "angular-mocks": "~1.3.0",
-    "patternfly": "git://github.com/patternfly/patternfly.git#ssui",
-    "angular-patternfly": "~2.0.0",
+    "patternfly": "~2.2.0",
+    "angular-patternfly": "~2.1.1",
     "angular-bootstrap": "~0.13.0",
     "angular-base64": "~2.0.5"
   },
   "resolutions": {
-    "angular": "~1.3.0",
-    "patternfly": "ssui"
+    "angular": "~1.3.0"
   },
   "overrides": {
     "patternfly": {
       "main": [
         "dist/js/patternfly.js",
+        "components/c3/c3.js",
```

Comment:
This shouldn't be required for this project. I will look to get this removed in angular-patternfly 2.2.0